core/src/main/scala/org/apache/spark/ui/storage/StorageTab.scala

```diff
@@ -60,7 +60,7 @@ class StorageListener(storageStatusListener: StorageStatusListener) extends Bloc
 
   override def onStageSubmitted(stageSubmitted: SparkListenerStageSubmitted): Unit = synchronized {
     val rddInfos = stageSubmitted.stageInfo.rddInfos
-    rddInfos.foreach { info => _rddInfoMap.getOrElseUpdate(info.id, info).name = info.name }
+    rddInfos.foreach { info => _rddInfoMap.getOrElseUpdate(info.id, info)}
```

Comment:
That's the question to you -- why _don't_  you just want to do that?. You're actually reverting the change in SPARK-9044 (use `git blame` to understand who changed this and when.).